### DISCUSSION TITLE
image shortcode

### DISCUSSION THREAD

**Discussion by TechieBlossom** [thuliteio/doks#186] [Q&A]
I am getting this error while using img shortcode and img simple shortcode. What it the solution to this?

execute of template failed: template: shortcodes/img.html:2:18: executing "shortcodes/img.html" at <$image.Resize>: nil pointer evaluating resource.Resource.Resize

**Comment by h-enk** [comment]
Are your referenced images a page resource /are you using page bundles? See also: [pages](https://getdoks.org/docs/recipes/pages/)

**Reply by TechieBlossom** [reply]
Params.Resources didn't work, but resources worked. Now, I can load images.

**Reply by TechieBlossom** [reply]
Thanks...

**Reply by NoneelPrasad** [reply]
Hi can you explain how you fixed loading of images?
I've been following the documentation and getting  nil pointer evaluating resource.Resource.Resize also.
